2022-11-21 21:30:09,813 INFO    MainThread:15684 [wandb_setup.py:_flush():68] Configure stats pid to 15684
2022-11-21 21:30:09,814 INFO    MainThread:15684 [wandb_setup.py:_flush():68] Loading settings from C:\Users\Brighton\.config\wandb\settings
2022-11-21 21:30:09,814 INFO    MainThread:15684 [wandb_setup.py:_flush():68] Loading settings from C:\git\Metadating\metabackend\api\wandb\settings
2022-11-21 21:30:09,814 INFO    MainThread:15684 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-21 21:30:09,814 WARNING MainThread:15684 [wandb_setup.py:_flush():68] Could not find program at -m __main__
2022-11-21 21:30:09,814 INFO    MainThread:15684 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': None, 'program': '-m __main__'}
2022-11-21 21:30:09,814 INFO    MainThread:15684 [wandb_init.py:_log_setup():476] Logging user logs to C:\git\Metadating\metabackend\api\wandb\run-20221121_213009-ft-OW0BlTRKUItf6mVuG2sdNsMz\logs\debug.log
2022-11-21 21:30:09,814 INFO    MainThread:15684 [wandb_init.py:_log_setup():477] Logging internal logs to C:\git\Metadating\metabackend\api\wandb\run-20221121_213009-ft-OW0BlTRKUItf6mVuG2sdNsMz\logs\debug-internal.log
2022-11-21 21:30:09,814 INFO    MainThread:15684 [wandb_init.py:init():516] calling init triggers
2022-11-21 21:30:09,814 INFO    MainThread:15684 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {'object': 'fine-tune', 'id': 'ft-OW0BlTRKUItf6mVuG2sdNsMz', 'hyperparams': <OpenAIObject at 0x2b1b2f3b950> JSON: {
  "batch_size": 8,
  "learning_rate_multiplier": 0.2,
  "n_epochs": 4,
  "prompt_loss_weight": 0.01
}, 'organization_id': 'org-fxJYI8WdQnGDT3Fdd4xfj9Hk', 'model': 'ada', 'training_files': <File file id=file-6qWfTBCH7ycyVAC4uruqpZZY at 0x2b1b2f3b9f0> JSON: {
  "bytes": 2546078,
  "created_at": 1669083156,
  "filename": "D:\\data\\metadating\\train.jsonl",
  "id": "file-6qWfTBCH7ycyVAC4uruqpZZY",
  "object": "file",
  "purpose": "fine-tune",
  "status": "processed",
  "status_details": null
}, 'validation_files': [], 'result_files': <File file id=file-bcqnmXE0OWNqjNCa4tAj4AP9 at 0x2b1b2f3ba90> JSON: {
  "bytes": 166518,
  "created_at": 1669084156,
  "filename": "compiled_results.csv",
  "id": "file-bcqnmXE0OWNqjNCa4tAj4AP9",
  "object": "file",
  "purpose": "fine-tune-results",
  "status": "processed",
  "status_details": null
}, 'created_at': datetime.datetime(2022, 11, 21, 21, 12, 36), 'updated_at': 1669084156, 'status': 'succeeded', 'fine_tuned_model': 'ada:ft-jasincase-2022-11-22-02-29-15'}
2022-11-21 21:30:09,815 INFO    MainThread:15684 [wandb_init.py:init():569] starting backend
2022-11-21 21:30:09,815 INFO    MainThread:15684 [wandb_init.py:init():573] setting up manager
2022-11-21 21:30:09,819 INFO    MainThread:15684 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn, using: spawn
2022-11-21 21:30:09,830 INFO    MainThread:15684 [wandb_init.py:init():580] backend started and connected
2022-11-21 21:30:09,839 INFO    MainThread:15684 [wandb_init.py:init():658] updated telemetry
2022-11-21 21:30:09,869 INFO    MainThread:15684 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-11-21 21:30:10,145 INFO    MainThread:15684 [wandb_run.py:_on_init():2000] communicating current version
2022-11-21 21:30:10,275 INFO    MainThread:15684 [wandb_run.py:_on_init():2004] got version response 
2022-11-21 21:30:10,275 INFO    MainThread:15684 [wandb_init.py:init():728] starting run threads in backend
2022-11-21 21:30:10,436 INFO    MainThread:15684 [wandb_run.py:_console_start():1980] atexit reg
2022-11-21 21:30:10,436 INFO    MainThread:15684 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2022-11-21 21:30:10,436 INFO    MainThread:15684 [wandb_run.py:_redirect():1903] Wrapping output streams.
2022-11-21 21:30:10,436 INFO    MainThread:15684 [wandb_run.py:_redirect():1925] Redirects installed.
2022-11-21 21:30:10,437 INFO    MainThread:15684 [wandb_init.py:init():765] run started, returning control to user process
2022-11-21 21:30:17,248 WARNING MsgRouterThr:15684 [router.py:message_loop():77] message_loop has been closed
